_templates/Additional_sidebar.html

```diff
@@ -0,0 +1,31 @@
+<h1>Welcome To my page </h1>
+<body><h2>My name is Enmanuel Morillo </h2>
+  <p>I'm majoring in <em>COMPUTER SCIENCE !</em> </body>
```

Comment:
```suggestion
<h2>My name is Enmanuel Morillo </h2>
  <p>I'm majoring in <em>COMPUTER SCIENCE !</em> </p>
```

this is inside of the body of the page already, so you don't need that tag.  and you opened the `p` without closing it

Reply:
fixed it